public/style.css

```diff
@@ -15,3 +28,57 @@
 ul {
```

Comment:
Since our `ul` element has an id right now(would be changed by class), we can move list-style property into related id/class.